userguide/virtualmachines.rst

```diff
@@ -52,6 +52,15 @@ shows a list of installed virtual machines.
 
 The |ui-options| menu has options for controlling and modifying VMs:
 
+* :guilabel:`Start` boots a VM. Alternatively, start a VM by clicking the
```

Comment:
Please avoid the "Alternatively," halting start to a sentence.  How about
"VMs can also be started by..."